tasks/collect-slack-notification-params/README.md

```diff
@@ -0,0 +1,13 @@
+# collect-slack-notification-params
+
+Tekton task that collects slack configuration options from Release CRs and the data file. A formatted message is emitted
```

Comment:
Looks like it just collects them from the data file, not Release CRs too

Reply:
Actually, I think the Parameters section in the README is just missing a bunch

Reply:
README fixed